hermes-2.cxx

```diff
@@ -252,7 +252,8 @@ int Hermes::init(bool restarting) {
   j_par = optsc["j_par"]
               .doc("Parallel current:    Vort <-> Psi")
               .withDefault<bool>(true);
-  
+
+  OPTION(optsc, j_pol_terms, true);
```

Comment:
Can we make the default `false` for now, at least until these new terms are working? I think they should be on by default eventually, but not yet...